### PR TITLE
Release 4.43.1

### PR DESCRIPTION
**Pull request by dnil**
This PR marks a new Scout release. We apply semantic versioning. This is a patch release since it patches an issue without changing the intended functionality or any interfaces.

### Added
 - [x] Added database index for OMIM disease term genes
 ### Changed
 ### Fixed
~Do not drop HPO terms collection when updating HPO terms via the command line~
- [x] Do not drop disease (OMIM) terms collection when updating diseases via the command line

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/commands/update/hpo.py [hunk 1]
```diff
@@ -43,8 +43,9 @@ def hpo(hpoterms, hpo_to_genes):
     adapter = store
 
     LOG.info("Dropping HPO terms")
-    adapter.hpo_term_collection.drop()
+    adapter.hpo_term_collection.delete_many({})
```
Comment (northwestwitch) [inline scout/commands/update/hpo.py:46]: You have incorporated also this change from my previous PR. But it should be fine when we merge #3016

**Review by northwestwitch — APPROVED** [review]
👍🏻

File: scout/commands/update/hpo.py (hunk 1 above)
Comment (dnil) [reply scout/commands/update/hpo.py]: Yes, that was intentional! All good!

**Comment by dnil** [comment]
This is now on stage for testing!

**Comment by dnil** [comment]
Adding index 🆗 
Screenshot 2021-12-03 at 09 53 10

**Comment by dnil** [comment]
Disease terms: 🆗 
Screenshot 2021-12-03 at 10 26 34

Screenshot 2021-12-03 at 10 27 33

As for HPO ahem, right, this is clever:
Screenshot 2021-12-03 at 10 21 28
And indexes are fine ofcourse:
Screenshot 2021-12-03 at 10 27 33

Non-dropping of indices was tested locally though in the respective PRs so I think we are fine.

**Pull request merged**